src/UserPreferences.py

```diff
@@ -0,0 +1,133 @@
+"""
+Class to access the API for the Database to Create directories
+for the storage of default user preferences
+"""
+import pathlib # finding and creating directories
+import os # getting environment attributes
+import sqlite3
+import subprocess # Running terminal code
+import logging # Logging what is happening in the code base
+
+from src.InterfaceUserPref import InterfaceUserPref
+from src.UserPreferencesDB import UserPreferencesDB # accessing the database
+
+logger = logging.getLogger(__name__) # Starting logger
+
+class UserPreferences(InterfaceUserPref):
+    """
+    To set and modify the default directory for the user to
+    begin on when opening the application
+    """
+    # For testing purposes:
+    #   True or Value means success
+    #   False or exception means did not succeed
+    # Unless otherwise Specified
+    def __init__(self, db_location=pathlib.Path.home()) -> None:
+        """ Creating dependent classes and setting up the file system for the user """
+        logger.info("START: UserPreferences Database")
+        db_location = pathlib.Path(db_location)
+        self.db_location = db_location/ ".onko" # database directory ( "." makes file hidden in linux and macOS)
```

Comment:
this suggests that what gets passed in is a directory.
See further in tests for followup comment.